src/lib/bootloader/grub2base.rb

```diff
@@ -148,6 +150,27 @@ def disable_serial_console
 
   private
 
+    def system_locale
+      begin
+        language = ::Bootloader::Language.new
+        language.load
+      rescue Errno::ENOENT
```

Comment:
I think this is the wrong place. CFA::BaseModel or CFA::AugeasParses should handle the exception (or document that the user must do it, which they don't).